setup.py

```diff
@@ -50,7 +50,6 @@
         'License :: OSI Approved :: MIT License',
 
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
```

Comment:
so far so good - but if testing is in place for py 3.7, should not it be added here as well?

Reply:
I will have another PR to add that